app/controllers/admin/admins_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::AdminsController < ApplicationController
+  before_action :require_admin
+
+  def show
+
+  end
+
+  private
+
+    def require_admin
+      render file: 'public/404' unless current_admin
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.